girder/api/v1/resource.py

```diff
@@ -207,6 +210,51 @@ def stream():
             yield zip.footer()
         return stream
 
+    @access.public(scope=TokenScope.DATA_READ)
+    @autoDescribeRoute(
+        Description('Download a resource based on its path')
+        .param('path', 'The path of the resource.', paramType='path')
```

Comment:
There's no documentation here on the use of this parameter (for instance, that you must prepend `'/user/'` or `'/collection'`).